tests/PhenX.EntityFrameworkCore.BulkInsert.Tests/DbContainer/TestDbContainer.cs

```diff
@@ -39,29 +38,42 @@ public async Task InitializeAsync()
         }
     }
 
-    public async Task DisposeAsync()
+    public async Task<TDbContext> CreateContextAsync()
```

Comment:
Are all your changes required for mysql provider to work ?

Reply:
Not really, I added a few more tests, but most of the stuff is needed because of the "reuse" to isolate the tests logically from each other. It could be also solved by creating a new database with each run, but I like this, because it is easier to debug. You can just restart the container and connect with pgadmin or other tools.

Reply:
Ok fair enough